srcpkgs/labwc/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'labwc'
 pkgname=labwc
-version=0.6.6
+version=0.7.0
 revision=1
 build_style=meson
 hostmakedepends="pkg-config scdoc wayland-devel"
-makedepends="cairo-devel glib-devel libpng-devel librsvg-devel libxml2-devel pango-devel wlroots0.16-devel"
+makedepends="cairo-devel glib-devel libpng-devel librsvg-devel libxml2-devel pango-devel wlroots0.17-devel"
 short_desc="Wayland window-stacking compositor"
 maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://labwc.github.io/"
 changelog="https://raw.githubusercontent.com/labwc/labwc/master/NEWS.md"
 distfiles="https://github.com/labwc/labwc/archive/refs/tags/${version}.tar.gz"
-checksum=fea73450a0cc806339e38414973261812f6376d09cf387e693a4bddf75c40916
+checksum=2607219cb766db4d258a20c62c98ea16cd6d62c735e65f35d916188cf0ccbe70
 
 post_install() {
```

Comment:
All of the files installed by `post_install` are already installed in `/usr/share/doc/labwc`. I think this function should just be dropped.